features/sp-solution-features/org.wso2.carbon.sp.solutions.apim.analytics.feature/src/main/resources/dashboard/apimanalytics.json

```diff
@@ -0,0 +1 @@
+{"dashboard":{"url":"apimanalytics","owner":"admin","name":"APIM Analytics","description":"Widgets to configure APIM Analytics Alert Configurations","landingPage":"home","parentId":"0","pages":[{"id":"home","name":"API Creator Alert Configuration","content":[{"type":"stack","isClosable":true,"title":"","content":[{"title":"APIM Api Creator Alert Configuration","type":"component","component":"APIMApiCreatorAlertConfiguration","props":{"id":"a6c4750a-0f9f-45da-9e44-5ac5b95529da","configs":{"pubsub":{"types":["publisher"]},"isGenerated":false,"options":{}},"widgetID":"APIMApiCreatorAlertConfiguration"},"isClosable":false,"header":{"show":true},"componentName":"lm-react-component"}]}]},{"id":"new-page-2","name":"API Subscriber Alert Configuration","content":[{"type":"stack","isClosable":true,"title":"","content":[{"title":"APIMApiSubscriberAlertConfiguration","type":"component","component":"APIMApiSubscriberAlertConfiguration","props":{"id":"18a6d20a-a8c9-44b8-8864-17fe9fe1be9f","configs":{"pubsub":{"types":["publisher"]},"isGenerated":false,"options":{}},"widgetID":"APIMApiSubscriberAlertConfiguration"},"isClosable":false,"header":{"show":true},"componentName":"lm-react-component"}]}]}],"hasOwnerPermission":false,"hasDesignerPermission":false,"hasViewerPermission":false},"widgets":{"generated":[],"custom":["APIMApiCreatorAlertConfiguration","APIMApiSubscriberAlertConfiguration"]}}
```

Comment:
Add new line EOF

Reply:
Fixed in a21aa920ceb5fdbd3f34fa9dddb75b5630876048 of https://github.com/wso2/analytics-apim.